modules/edu.gemini.giapi.client/src/main/scala/edu/gemini/giapi/client/package.scala

```diff
@@ -75,25 +117,66 @@ package client {
     * Interpreters
     */
   object Giapi {
+    private final case class StatusStreamer(aggregate: StatusHandlerAggregate, ss: StatusService)
+
+    private def statusGetter[F[_]: Sync](c: ActiveMQJmsProvider): F[StatusGetter] = Sync[F].delay {
+      val sg = new StatusGetter("client")
+      sg.startJms(c)
+      sg
+    }
+
+    private def commandSenderClient[F[_]: Applicative](c: ActiveMQJmsProvider): F[CommandSenderClient] =
+      Applicative[F].pure { new CommandSenderClient(c) }
```

Comment:
Does `CommandSenderClient` have any internal state? If so it needs to be in `delay`. Note that you can also say `(new CommandSenderClient(c)).pure[F]` here.

Reply:
It keeps a reference to `c` but dosen't do anything on the constructor

Reply:
Ok that's fine as-is.